packages/mdl-elevation/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "mdl-elevation",
+  "description": "SASS mixins + CSS Classes for adding Material Design elevation to elements",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "mdl-animation": "^1.0.0"
+  }
```

Comment:
The demo seems to depend on `mdl-typography`, so maybe add that in as a `devDependency`?